bigtable/integration_test.go

```diff
@@ -96,7 +103,46 @@ func TestMain(m *testing.M) {
 			c.Cluster, "us-central1-b", "1",
 		)
 	}
-	os.Exit(m.Run())
+	exit := m.Run()
+	if err := cleanup(c); err != nil {
+		log.Printf("Post-test cleanup failed: %v", err)
+	}
+	os.Exit(exit)
+}
+
+func cleanup(c IntegrationTestConfig) error {
+	// Cleanup resources marked with bt-it- after a time delay
+	timeToWait := time.Hour * 48 // two days
+	instancePrefix := "bt-it-"
```

Comment:
nit: could this be a const so the cleanup and create can stay in sync should one change over time.